scio-core/src/main/scala/com/spotify/scio/ScioContext.scala

```diff
@@ -446,12 +457,16 @@ class ScioContext private[scio] (val options: PipelineOptions,
   /**  Whether this is a test context. */
   def isTest: Boolean = testId.isDefined
 
-  private[scio] def testIn: TestInput = TestDataManager.getInput(testId.get)
```

Comment:
`TestInput` and `TestInputNio` were separated which caused existing tests to fail since they use `.input` but all the bqRead* methods are using nio now.